osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -263,6 +275,7 @@ public override bool OnExiting(IScreen next)
 
             buttons.State = ButtonSystemState.Exit;
             this.FadeOut(3000);
+            songTicker.Hide();
```

Comment:
nor should this

Reply:
And here It can be visible too

Reply:
That's why this exactly one is required
https://streamable.com/jecpp